solidity/ecdsa/deploy/06_deploy_wallet_registry_governance.ts

```diff
@@ -7,7 +7,8 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 
   const WalletRegistry = await deployments.get("WalletRegistry")
 
-  const GOVERNANCE_DELAY = 604800 // 1 week
+  // 60 seconds for Goerli. 1 week otherwise.
+  const GOVERNANCE_DELAY = hre.network.name === "goerli" ? 60 : 604800
```

Comment:
Not in this PR but I propose we do the same for the beacon for consistency:

https://github.com/keep-network/keep-core/blob/ff77e6eefdd9123da3f72388b1f470a444dd7346/solidity/random-beacon/deploy/07_deploy_random_beacon_governance.ts#L10